service/history/queues/alerts.go

```diff
@@ -33,12 +33,18 @@ import (
 type (
 	// Alert is created by a Monitor when some statistics of the Queue is abnormal
 	Alert struct {
-		AlertType                  AlertType
-		AlertAttributesReaderStuck *AlertAttributesReaderStuck
+		AlertType                            AlertType
+		AlertAttributesQueuePendingTaskCount *AlertAttributesQueuePendingTaskCount
```

Comment:
nit: remove Queue from the type name and variable name?

Reply:
Added `Queue` in the names since we may/will monitor # of pending tasks on a host level in the future, so want to avoid naming confusion when we do that.